coreclr-debug/NuGet.config

```diff
@@ -4,5 +4,7 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <!-- This dependency is not present in the release branch -->
+    <add key="coreclrdebug" value="https://www.myget.org/F/coreclr-debug/api/v3/index.json" />    
```

Comment:
I think you can commit this PR as-is and I will fix it once you are done. Alternatively I can fix it in master. Either way is fine with me.